frontend/.gcloudignore

```diff
@@ -0,0 +1,17 @@
+# This file specifies files that are *not* uploaded to Google Cloud Platform
```

Comment:
I think this file should already be in master? Can you try to rebase the branch to see if these diff are removed in the PR?

Reply:
I'll remove this file from this PR, but also I think we can take it out of master as well. I now realize that it's added every time the webapp is deployed

Reply:
I see. So you can add `frontend/.gcloudignore` or add `**/.gcloudignore` to `.gitignore` so future PR won't include it